omron_test_node:
  ros__parameters:
    use_close_loop: true
    cmd_vel_topic: "/cmd_vel"
    max_vel_in_m: 0.5 # m/s
    max_acc_in_m: 1.0 # m/s^2
    max_rot_vel_in_deg: 30.0 # deg/s
    max_rot_acc_in_deg: 300.0 # deg/s^2
    update_period_in_ms: 50 # ms
    movements: [0]
    values:    [2.0]

    alpha: 0.1
